src/main/java/org/sagebionetworks/web/client/widget/table/v2/schema/ColumnModelTableRowEditorWidgetImpl.java

```diff
@@ -260,12 +269,25 @@ private String getMaxSizeForType(ColumnTypeViewEnum type) {
     }
   }
 
+  Set<ColumnTypeViewEnum> unsupportedTypesForViews = new HashSet<ColumnTypeViewEnum>(
+    Arrays.asList(ColumnTypeViewEnum.LargeText, ColumnTypeViewEnum.MediumText)
+  );
+
   @Override
   public void configure(
     ColumnModel model,
     SelectionPresenter selectionPresenter
   ) {
     view.setSelectionPresenter(selectionPresenter);
+
+    // add column types to the dropdown:
+    Arrays
+      .asList(ColumnTypeViewEnum.values())
+      .stream()
+      .filter((val -> (!isView || !unsupportedTypesForViews.contains(val)))) // keep all values for tables, or keep value if not found in unsupportedTypesForViews.
+      .forEach(val -> {
+        view.addColumnType(val);
```

Comment:
https://sagebionetworks.jira.com/browse/SWC-6333

Reply:
Removed the list from the ui.xml, and add dynamically.  If View, do not include MediumText or LargeText